src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,43 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private int capacity;
+
+    private String description;
+
+    public CinemaHall() {
+
+    }
+
+    public CinemaHall(int capacity, String description) {
+        this.capacity = capacity;
+        this.description = description;
+    }
+
+    public void setCapacity(int capacity) {
+        this.capacity = capacity;
+    }
+
+    @Override
```

Comment:
It's recommended to implement `equals` and `hashCode` methods for entities. Hibernate uses these methods to determine if the entity is the same or different when it comes to entity comparison and caching.